roles/rsyslog/templates/general_action_params.j2

```diff
@@ -0,0 +1,7 @@
+{% if __rsyslog_output.action is defined and __rsyslog_output.action is mapping %}
+{%   for option, value in __rsyslog_output.action.items() %}
+{%     if option | lower in rsyslog_valid_general_action_params %}
+action.{{ option }}="{{ value }}"
```

Comment:
some of the action parameters have types other than `string` - e.g. https://www.rsyslog.com/doc/v8-stable/configuration/actions.html#general-action-parameters
Some of them are integers - is it ok to have `action.resumeInterval="30"`?  or does it have to be `action.resumeInterval=30`?  I guess the same applies to the queue parameters below.

Reply:
Fair, I guess a better approach would be 

{% if __rsyslog_output.action is defined and __rsyslog_output.action is mapping %}
{%   for option, value in __rsyslog_output.action.items() %}
{%     if option | lower in rsyslog_valid_general_action_params and value is number %}
action.{{ option }}={{ value }}
{%     elif option | lower in rsyslog_valid_general_action_params  %}
action.{{ option }}="{{ value }}"

Reply:
Looks like `action.resumeInterval="30"` is fine - https://github.com/rsyslog/rsyslog/blob/master/tests/es-basic-abort.sh#L19-L20

```
		action.resumeRetryCount="-1"
		action.resumeInterval="1"
```

so, sorry for the noise - your approach looks good

Reply:
Or split it in to two dicts

{% if __rsyslog_output.action is defined and __rsyslog_output.action is mapping %}
{% for option, value in __rsyslog_output.action.items() %}
{% if option | lower in rsyslog_valid_general_action_params_as_int %}
action.{{ option }}={{ value }}
{% elif option | lower in rsyslog_valid_general_action_params_as_string %}
action.{{ option }}="{{ value }}"